package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "mercadopago",
-  "version": "2.1.1",
+  "version": "2.1.2",
```

Devemos subir a minor, porque estamos adicionando um campo novo.
```suggestion
  "version": "2.2.0",
```